dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Patient/Index.cshtml

```diff
@@ -54,25 +51,43 @@
                     <tr>
                         <td class="ms-thumbnail col-md-1"><img class="img-thumbnail" src="@task.TaskImageUrl" alt="Task thumbnail icon" /></td>
                         <td class="col-md-9 @(task.ScheduleType == TimelineScheduleType.Unscheduled ? "occurrence" : "")">
-                            @task.TaskName<br />
-                            @TaskTime(task)
+                            @if (task.ScheduleType == TimelineScheduleType.Unscheduled)
+                            {
+                                @task.TaskName
+                            }
+                            else
+                            {
+                                @Html.ActionLink(task.TaskName, "TaskOccurrence", new {id = task.TaskId, startDate = Model.StartDate, endDate = Model.EndDate, personId = Request.Params["personId"], recordId = Request.Params["recordId"]})
```

Comment:
In part with question on the controller, I struggle to understand what `StartDate` and `EndDate` mean on a task occurrence. I didn't think we had a notion of that, or at least I'm struggling to interpret/remember it.